### PR TITLE
promisify and return

### PR DESCRIPTION
**Pull request by fippo**
applying the codemod as well. Should fix https://github.com/otalk/RTCPeerConnection/pull/69

### CODE REVIEW

**Comment by fippo** [comment]
just remembered #10 --are you using a codemod too @xdumaine?

**Comment by xdumaine** [comment]
@fippo No, I did it manually. I like yours better, which changes to use promise syntax as well, except I'd change the catches, too.

File: index.js [hunk 1]
```diff
@@ -160,39 +164,43 @@ TraceablePeerConnection.prototype.close = function () {
 TraceablePeerConnection.prototype.createOffer = function (successCallback, failureCallback, constraints) {
     var self = this;
     this.trace('createOffer', constraints);
-    this.peerconnection.createOffer(
+    return this.peerconnection.createOffer(
+        constraints
+    ).then(
```
Comment (xdumaine) [inline index.js:169]: These should have matching `catch` instead of passing both functions to `then`.

Comment (fippo) [inline index.js:169]: no, see https://github.com/webrtc/samples/pull/789#discussion_r62529798

**Comment by fippo** [comment]
published as 1.2.0

**Pull request merged**